src/main/java/com/mycompany/exercisethree/Calculator.java

```diff
@@ -0,0 +1,82 @@
+package com.mycompany.exercisethree;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+/**
+ *
+ * @author Cristian Gallego
+ */
+public class Calculator {
+
+    public List<Double> history = new ArrayList<>();
+    public List<Double> listNumbers = new ArrayList<>();
+    public int option;
+
+    public List<Double> getInputString() {
+        Scanner scanner = new Scanner(System.in);
+        System.out.println("Write the numbers:");
+        String values = scanner.nextLine();
+        List<Double> numbersList = new ArrayList<>();
+        Pattern pattern = Pattern.compile("\\d+");
```

Comment:
You can have the regex as constant in java https://www.javatpoint.com/java-constant

Reply:
You mean this:

`    final Pattern pattern = Pattern.compile("\\d+");
        final Matcher matcher = pattern.matcher(values);`